src/sentry/receivers/savedsearch.py

```diff
@@ -3,47 +3,43 @@
 from django.db.models.signals import post_save
 
 from sentry.models import Project, SavedSearch
+from sentry.signals import save_search_created
+
+
+DEFAULT_SAVED_SEARCHES = [
+    {'name': 'is:unresolved', 'query': 'is:unresolved', 'is_default': True},
+    {'name': 'Needs Triage', 'query': 'is:unresolved is:unassigned'},
+    {'name': 'Assigned To Me', 'query': 'is:unresolved assigned:me'},
+    {'name': 'My Bookmarks', 'query': 'is:unresolved bookmarks:me'},
+    {'name': 'New Today', 'query': 'is:unresolved age:-24h'},
+]
 
 
 def create_default_saved_searches(instance, created=True, **kwargs):
     if not created:
         return
 
-    SavedSearch.objects.create(
-        project=instance,
-        name='Unresolved Issues',
-        query='is:unresolved',
-        is_default=True,
-    )
-
-    SavedSearch.objects.create(
-        project=instance,
-        name='Needs Triage',
-        query='is:unresolved is:unassigned',
-    )
-
-    SavedSearch.objects.create(
-        project=instance,
-        name='Assigned To Me',
-        query='is:unresolved assigned:me',
-    )
-
-    SavedSearch.objects.create(
-        project=instance,
-        name='My Bookmarks',
-        query='is:unresolved bookmarks:me',
-    )
-
-    SavedSearch.objects.create(
-        project=instance,
-        name='New Today',
-        query='is:unresolved age:-24h',
-    )
+    for search_kwargs in DEFAULT_SAVED_SEARCHES:
+        search_kwargs.update({'project': instance})
```

Comment:
@tkaemming wouldn't this be mutating the original object inside of `DEFAULT_SAVED_SEARCHES`?

Would be safer to just do like:

```python
SavedSearch.objects.create(project=instance, **search_kwargs)
```